File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include  <stdio.h>

int main(void)
{
    int n; scanf("%d",&n);
    int max=-1000000;
    int min=1000000;
    for(int i=0;i<n;i++){
        int a; scanf("%d",&a);
        if(a>max) max=a;
        if(a<min) min=a;
    }
    printf("%d %d",min,max);
    return 0;
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int max=-1000000;
		int min=1000000;
		for(int i=0; i<n; i++) {
			int a = sc.nextInt();
			if(a>max) max=a;
			if(a<min) min=a;
		}
		System.out.print(min+" "+max);
	}

}
